File: broken-sequence/broken-sequence.py
def find_missing_number(sequence):
    try:
        if len(sequence)==0:    
            return 0
        t = [int(i) for i in sequence.split()]
        t.sort()
        for i in range(1,max(t)+1):
            if i not in t:
                return i
        return 0
    except:
        return 1

File: broken-sequence/broken-sequence.js
function findMissingNumber(sequence){
  if(sequence==""){return 0;};
  var t = sequence.split(" ").map(function(num){
  return parseInt(num);
});;
  var j = t.reduce(function(a,b){
  return a+b;
})
  if (isNaN(j)){return 1;}
  var n = Math.max(...t);
  for(var i=1;i<n;i++){
  if(t.indexOf(i)==-1){return i;};
  };
  return 0;
}